lib/util/json-log-humanizer.js

```diff
@@ -53,6 +48,19 @@ humanizer.processLine = function(line) {
   }
 };
 
+function extraData(data) {
+  var extras = "";
+  if (data.timestamp) {
+    extras += " timestamp: " + data.timestamp;
```

Comment:
Normally I'd expect timestamp to be something that goes on the front of the line rather than additional data.

Like an nginx log for example:
```
2010/08/23 15:25:35 [error] 19997#0: *1 open() "/var/www/nginx-defau...
```